irohad/ametsuchi/temporary_wsv.hpp

```diff
@@ -51,7 +51,7 @@ namespace iroha {
        */
       virtual bool apply(const model::Transaction &transaction,
                          std::function<bool(const model::Transaction &,
-                                            CommandExecutor &, WsvQuery &)>
+                                            WsvCommand &, WsvQuery &)>
                              function) = 0;
```

Comment:
`  if (cmd.validate(WsvQuery, Context)){`
Is this `Context` `TemporaryWsv`?

`cmd.execute(WsvQuery, WsvCommand, Context);`
Is this `Context` `MutableStorage`?